frontend/lib/store/RegattaStore.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:async';
+import 'package:angular2/core.dart';
+import 'package:greencat/greencat.dart';
+import 'package:frontend/models/event.dart';
+import 'package:frontend/services/event_service.dart';
+import 'package:frontend/app_component.dart';
+
+enum ActionType { addEvent, addNewEvent }
+
+/// Actions to be triggered to the app store.
```

Comment:
Yes they are doc comments, see: https://www.dartlang.org/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types